go/vt/mysqlctl/mysqlshellbackupengine_test.go

```diff
@@ -406,6 +406,10 @@ func TestMySQLShellBackupEngine_ExecuteBackup_ReleaseLock(t *testing.T) {
 
 		require.Equal(t, mysqlShellBackupEngineName, manifest.BackupMethod)
 
+		if hostname, err := os.Hostname(); err == nil {
+			require.Equal(t, hostname, manifest.Hostname)
+		}
+
```

Comment:
the `mysqlshell` engine is the only one we have unit tests for `ExecuteBackup`, so I only verified this here